Executor/JobExecutor.php

```diff
@@ -263,19 +263,26 @@ public function execute(string $code, ?OutputInterface $output = null)
 
         /** @var string[] $entities */
         $entities = explode(',', $code);
-        if (count($entities) > 1) {
-            $entities = $this->sortJobs($entities);
+        $entities = $this->sortJobs($entities);
 
-            /** @var string $jobCode */
-            foreach ($entities as $jobCode) {
-                $this->execute($jobCode, $output);
-            }
-
-            return true;
+        /** @var string $jobCode */
+        foreach ($entities as $jobCode) {
+            $this->runImport($code);
         }
 
         $this->output = $output;
+        $this->eventManager->dispatch('akeneo_connector_all_imports_finished');
```

Comment:
Could you add a check if `$entities` is not empty?
If you take a look at `Cron/LaunchScheduledJob` you can see that this is called every minute, even when there are no jobs scheduled.

Or add the check in the cron